src/components/footer/Footer.jsx

```diff
@@ -61,7 +77,7 @@ const Footer = () => {
           Made with <i className="fas fa-heart"></i> by Apurva Sawant
```

Comment:
You can use react icons, they are much better and in that you get all icons of different sites in one.

Reply:
I tried react icons, but the size of each icon was differing. I'll try to add those when I refactor my code